apps/web/src/modules/create-dao/components/Artwork/ArtworkForm.tsx

```diff
@@ -1,10 +1,10 @@
 import { Button, Flex } from '@zoralabs/zord'
-import { Form, Formik } from 'formik'
+import { Field, FieldProps, Form, Formik } from 'formik'
 import isEmpty from 'lodash/isEmpty'
 import React from 'react'
 
-import TextArea from 'src/components/Fields/TextArea'
 import { Icon } from 'src/components/Icon'
+import { MarkdownEditor } from 'src/modules/create-proposal/components/ReviewProposalForm/MarkdownEditor'
```

Comment:
Perhaps we refactor MarkdownEditor into a more general directory sense it is being reused outside of the `ReviewProposalForm`

Reply:
Yep, that makes sense.